bzl/host/BUILD.bazel

```diff
@@ -0,0 +1,29 @@
+package(default_visibility = ["//visibility:public"])
+
+config_setting(
+    name = "linux",
+    constraint_values = [
+        "@platforms//os:linux",
+    ]
+)
+
+config_setting(
+    name = "linux_arm",
```

Comment:
I'm not sure that we support ARM at the moment (and don't have an ARM machine to test on). Have you tested on ARM? If not, I think we should remove this or comment it out, to make sure we're not giving an impression of ARM support where it doesn't necessarily exist.